src/test/groovy/com/palantir/gradle/revapi/RevapiSpec.groovy

```diff
@@ -980,6 +982,7 @@ class RevapiSpec extends IntegrationSpec {
     }
 
     @RestoreSystemProperties
+    @Ignore("doesn't work with latest Gradle version and needs to be fixed")
```

Comment:
Should we have a PR to fix it first and then bump the gradle version or is it dependent on this change?

Reply:
it started to fail with the Gradle version bump. My guess is that the test itself needs some slight configuration changes to work with the latest Gradle version (but it shouldn't block bumping Gradle)

Reply:
Ack. Later we can have an issue to track this.